docs/build.md

```diff
@@ -47,3 +91,9 @@ For any targets requiring ArborX, add
 ```CMake
 target_link_libraries(<target> ArborX::ArborX)
 ```
+Note that because of the way CMake deals with dependencies, users of ArborX
+will need to make sure that Kokkos installation directory is visible to CMake.
+The easiest way to address it is to add both to configuration options:
```

Comment:
If anyone knows a solution to this problem, I would be happy to hear it.

Reply:
Actually Cabana does this https://github.com/ECP-copa/Cabana/blob/27a31ab987d978eff18b443de1f5dc658c02237e/core/src/CabanaConfig.cmakein#L2-L4

Reply:
Not endorsing it, just sharing

Reply:
I'm don't understand that code and it's relevance here. Where is that `Kokkos_DIR` in the check coming from?

Reply:
Say you are configuring a code that does `find_package(Cabana)`.  Unless you set `Kokkos_DIR` yourself, Cabana's exported configuration sets the variable for you which guarantees the same Kokkos that was used to build Cabana is found.

Reply:
Continued in #287.